Source/JavaScriptCore/b3/B3Common.cpp

```diff
@@ -70,10 +70,10 @@ bool shouldSaveIRBeforePhase()
     return Options::verboseValidationFailure();
 }
 
-std::optional<GPRReg> pinnedExtendedOffsetAddrRegister()
+std::optional<GPRReg> linkRegister()
```

Comment:
I would keep mostly the same as the old name, but no need to say `pinned` anymore. Maybe just `extendedOffsetAddrRegister`

Reply:
updated